activerecord/lib/active_record/associations/collection_proxy.rb

```diff
@@ -715,21 +715,24 @@ def distinct
       end
       alias uniq distinct
 
-      # Count all records using SQL.
+      # Count all records.
       #
       #   class Person < ActiveRecord::Base
       #     has_many :pets
       #   end
       #
       #   person.pets.count # => 3
+      #   # This will perform the count using SQL.
       #   person.pets
       #   # => [
       #   #       #<Pet id: 1, name: "Fancy-Fancy", person_id: 1>,
       #   #       #<Pet id: 2, name: "Spook", person_id: 1>,
       #   #       #<Pet id: 3, name: "Choo-Choo", person_id: 1>
       #   #    ]
-      def count(column_name = nil)
-        @association.count(column_name)
+      #   person.pets.count { |pet| pet.name.include?('-') } # => 2
+      #   # Passing a block will select all pets in SQL and perform the count using Ruby.
```

Comment:
Okay. I was trying to follow the conventions of the project. If you look just below, there is an example of the explanatory comment going one line below the code example (line 747).